frame/staking/src/pallet/impls.rs

```diff
@@ -1432,13 +1406,31 @@ impl<T: Config> SortedListProvider<T::AccountId> for UseValidatorsMap<T> {
 	fn contains(id: &T::AccountId) -> bool {
 		Validators::<T>::contains_key(id)
 	}
+	fn get_score(id: &T::AccountId) -> Result<Self::Score, Self::Error> {
+		Ok(Pallet::<T>::weight_of(id).into())
+	}
+	fn try_state() -> Result<(), &'static str> {
+		Ok(())
+	}
+
+	frame_election_provider_support::runtime_benchmarks_or_test_enabled! {
```

Comment:
I don't see this defined anywhere. And I am a bit skeptical it is less characters than just typing `#[cfg(any(feature = "runtime-benchmarks", test))]` 😄

Reply:
It's defined in here: https://github.com/paritytech/substrate/blob/74a2a48e4ea4cb855311163752c834470b867a14/frame/election-provider-support/src/lib.rs#L688

And it does not work with `[cfg]`, hence the point of defining it.